### PR TITLE
Feature seeding script

### PR DESCRIPTION
**Pull request by mariomui**
Please review my seeding script.
requires mongoose and faker.

### CODE REVIEW

File: scripts/normalbatcher.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
+/* eslint-disable no-loop-func */
+// const puppeteer = require('puppeteer');
+
+// let scrape = async () => {
```
Comment (johnsonkwok) [inline scripts/normalbatcher.js]: Please remove any commented out code if it's not needed.

File: .gitignore [hunk 2]
```diff
@@ -24,3 +24,4 @@ coverage
 build/
 dist/
 example/
+proxy-server/
```
Comment (johnsonkwok) [inline .gitignore]: Should this proxy-server folder be in this repo?

**Comment by johnsonkwok** [comment]
Please write at least one unit test that passes. It is a requirement for pull requests.

File: scripts/normalbatcher.js (hunk 1 above)
Comment (mariomui) [reply scripts/normalbatcher.js]: Thanks!

**Pull request closed without merging**